### DISCUSSION TITLE
Is it possible to configure dynamic content ?

### DISCUSSION THREAD

**Discussion by jojomillennium** [directus/directus#2973] [Feature Requests]
Is it possible to configure Dynamic content the same way Strapi does?

**Comment by rijkvanzanten** [comment]
Heya! Could you share some specific examples? What's your exact use case?

**Comment by jojomillennium** [comment]
Sure!
I want to be able to decide for a page's content to have first a Header component, then a Slider of picture, then WYSIWYG, then Testimonial, then WYSIWYG.
but for another page I would like to have first a WYSIWYG, then slider, then WYSIWYG.
Kind of find the "component" aspect of a CMS.

I don't want to have to create specific collection singleton for each page, otherwise I can't let the "power" to the editors to manage the content of their pages. I'd like more of a "pages" collection, with fields allowing me to select which component and in which order.
I've tried to play around with relationship without success.

For example:
I'd create a collection "Media" with as field a `title`, an `image`, a `description`
a Collection "Slider" with a O2M to Media so I can add multiple image to it
A Collection "Testimonial" with fields `name`, `picture`, `quote`, `date`
A Collection Text with field `text`

Then, my Collection "Pages" (for my editors), and have some kind of dynamic field, that would be a O2M kind but to multiple collections, so I can select from other collections so to add a Slider, a Text, a Testimonial, a Text (again), etc...

Makes sense?

**Reply by rijkvanzanten** [reply]
Yup! We call that a "many to any" relationship, and has been discussed heavily over the last couple months 😄 

The API actually supports this "one field can relate to multiple other collections" already, and we're finishing up some final design ideas and agreements before jumping in to dev. So it's coming up!

**Reply by jojomillennium** [reply]
Nice! looking forward to test it!

**Comment by rijkvanzanten** [comment]
This has been implemented! Available starting in `v9.0.0-rc.19`

**Reply by srchulo** [reply]
Is this release cut not published yet?

**Reply by jojomillennium** [reply]
It's there, I've tested it using M2A relationship

**Reply by nithiz** [reply]
Works pretty good! I just can't seem to add another related collection to the M2A field once i saved the configuration. All fields are disabled in the "Relationship" tab. Is this a bug or am i using it wrong?

**Comment by rijkvanzanten** [comment]
This has since been implemented through the M2A relationship type!
